outputs/cleanup-workflows.sh

```
@@ -0,0 +1,126 @@
+mkdir -p .github/workflows.disabled
+
+# Workflows to disable (move to .disabled)
+WORKFLOWS_TO_DISABLE=(
+    "ci-cd.yml"
+    "ci.yml"
+    "type-safe-ci.yml"
+    "kimi-enhancer.yaml"  # Duplicate of .yml
+)
+
+# Workflows to keep but optimize
+WORKFLOWS_TO_OPTIMIZE=(
```

Shellcheck (0.11.0)

[warning] WORKFLOWS_TO_OPTIMIZE appears unused. Verify use (or export if used externally).

(SC2034)